src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java

```diff
@@ -77,7 +78,7 @@ public static void registerPowerUnitRecipes() {
 
                 ModHandler.addShapedEnergyTransferRecipe(recipeName, powerUnitStack,
                         Ingredient.fromStacks(batteryStack), false,
-                        "S  ", "GMG", "PBP",
+                        "S d", "GMG", "PBP",
```

Comment:
whats the `d` for?

Reply:
The `d` adds a screwdriver to the recipe, which should have been there originally, but was forgotten in GTCE.

Reply:
fair